source/proximity_list/proximitylist_api.cpp

```diff
@@ -149,13 +152,14 @@ void find(navitia::PbCreator& pb_creator,
             // we sort the proximity list on the indexes
```

Comment:
For a places nearby we want the 10 closest, not the 10 with lowest index, which is what we would get, no?